### PR TITLE
8300 configuration mask select contact form

### PR DESCRIPTION
**Pull request by hungnc89**
configuration mask select contact form

### CODE REVIEW

**Review by jayay — CHANGES_REQUESTED** [review]
Thanks for the work!

Most of it is good, but I'm sure the migration step would break existing sites. Please have a look at my comments and feel free to ask me.
Other than that, I've written a couple of comments right here.

File: plugin/EstateDetail.php [hunk 1]
```diff
@@ -109,6 +109,23 @@ public function getEstateMovieLinks(): array
 		return $result;
 	}
 
+	/**
+	 *
+	 * @return array Returns an string
```
[the remark this answers is not in the document]
Comment (jayay) [reply plugin/EstateDetail.php]: The method can be shortened all the way:

```php
<?php
public function getShortCodeForm(): string
{
	return  '[oo_form form="' . $this->getDataView()->getShortCodeForm(). '"]';
}
```

File: plugin/Installer/DatabaseChanges.php [hunk 2]
```diff
@@ -309,6 +347,68 @@ private function getCreateQueryFormFieldConfig()
 		return $sql;
 	}
 
+	/**
+	 *
+	 */
+
+	private function installDataQueryFormFieldConfig($defaultFormId)
```
Comment (jayay) [inline plugin/Installer/DatabaseChanges.php:354]: A type hint would be nice

**Review by jayay — APPROVED** [review]
It looks good to me. Unfortunately, we're going to have to wait for #29 to be merged first because `DatabaseChanges.php` will conflict.

**Comment by jayay** [comment]
@hungnc89 Would you please rebase this on top of master? This is going to be released soon

**Comment by jayay** [comment]
I made a new branch for the fixes, it is [8300-configuration-mask-select-contact-form-2](/onOfficeGmbH/oo-wp-plugin/tree/8300-configuration-mask-select-contact-form-2). However, there is a risky test (it has been there before, but our unit test configuration changed), which makes the tests fail. Would you please have a look?

**Comment by hungnc89** [comment]
> I made a new branch for the fixes, it is [8300-configuration-mask-select-contact-form-2](/onOfficeGmbH/oo-wp-plugin/tree/8300-configuration-mask-select-contact-form-2). However, there is a risky test (it has been there before, but our unit test configuration changed), which makes the tests fail. Would you please have a look?

**Comment by jayay** [comment]
Thank you!

**Pull request merged**